scalafmt-tests/src/test/resources/scala3/InlineTransparent.stat

```diff
@@ -0,0 +1,22 @@
+maxColumn = 20
```

Comment:
can you add tests so that lines would need to bed broken around these new modifiers, and see see how the code is formatted? indentation is interesting.

Reply:
I added an additional test, but the modifiers don't seem to be ever broken into lines.